src/control/lib/daos/ddb_types.go

```diff
@@ -0,0 +1,27 @@
+// (C) Copyright 2023 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+package daos
+
+import (
+	"github.com/google/uuid"
+)
+
+// CommandContext structure used for all commands
+type CommandContext struct {
+	DdbContext        interface{}
+	JsonOutput        bool
+	JsonOutputHandled bool
+}
+
+// SuperBlock The pool super block used for the superblock_dump command
+type SuperBlock struct {
```

Comment:
minor: To avoid conflict with other types of superblocks (e.g. the control plane superblock), it would be clearest to name this `PoolSuperblock`. Also, `superblock` is a single word, so `Superblock` instead of `SuperBlock` would be preferred.